packages/dashboard-backend/src/index.ts

```diff
@@ -85,7 +85,7 @@ server.get(
 );
 
 server.post(
-  "/workspace",
+  "/namespace/:namespace/devworkspaces",
```

Comment:
If you want to use namespace like that you'll need to do something like: https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-backend/src/index.ts#L78 (unless we just ignore namespace here)

Reply:
thanks for point it. I assumed I can draft paths and let someone else fix the logic haha.
I'll fix it before merging.